src/proc.ts

```diff
@@ -74,6 +72,14 @@ export interface ExecutionOptions {
   cwd?: string;
 }
 
+export interface ExecFileOptions {
```

Comment:
You can have this interface extend `ExecutionOptions`, just to remove duplication